tests/openadapt/test_scrub.py

```diff
@@ -0,0 +1,181 @@
+"""Module to test scrub.py"""
+from io import BytesIO
+import os
+from PIL import Image
+import pytesseract
+from openadapt.scrub import scrub, scrub_image
+
+
+def test_scrub_image_data() -> None:
+    """
+    Test is to be sure that the scrubbed image data is different
+    """
+    # Read test image data from file
+    test_image_path = "assets/test_scrub_image.png"
+    with open(test_image_path, "rb") as file:
+        test_image_data = file.read()
+
+    # Convert image data to PIL Image object
+    test_image = Image.open(BytesIO(test_image_data))
+
+    # Scrub the image
+    scrubbed_image = scrub_image(test_image)
+
+    # Save the scrubbed image data to a file
+    scrubbed_image_path = "scrubbed_image.png"
+    scrubbed_image.save(scrubbed_image_path)
+
+    # Load the scrubbed image from file for manual verification
+    scrubbed_image = Image.open(scrubbed_image_path)
+
+    # Perform OCR on the scrubbed image
+    ocr_text = pytesseract.image_to_string(scrubbed_image)
+
+    test_image.close()
+    scrubbed_image.close()
+    os.remove(scrubbed_image_path)
+
+    assert "krish@openadapt.ai" not in ocr_text
+    assert "Manage your Google Account" in ocr_text
+    assert "Sign out" in ocr_text
+
+
+def test_empty_string() -> None:
+    """
+    Test is to ensure that an empty string is returned
+    if an empty string is passed to the scrub function.
+    """
+    text = ""
+    expected_output = ""
+    assert scrub(text) == expected_output
+
+
+def test_no_scrub_string() -> None:
+    """
+    Test is to ensure that the same string is returned
```

Comment:
From https://stackoverflow.com/questions/59902102/why-is-imperative-mood-important-for-docstrings:
> First line should be in imperative mood.

@Mustaballer can we please use pydocstyle?

Reply:
Done for test_scrub.py